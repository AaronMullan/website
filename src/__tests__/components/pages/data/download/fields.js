import React from 'react'
import renderer from 'react-test-renderer'
import mockJSON from 'sample-openapi'
import Fields from '~components/pages/data/download/fields'

jest.mock('../../../../../../_api/v1/openapi.json', () => mockJSON)

describe('Components : Pages : Data : Download : Fields', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Fields schema="States" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
