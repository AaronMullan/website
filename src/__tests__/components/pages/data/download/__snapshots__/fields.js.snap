// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Data : Download : Fields renders correctly 1`] = `
<dl
  className="fields"
>
  <div>
    <dt>
      totalTestResults
    </dt>
    <dd>
      <div
        className="type"
      >
        <span
          className="a11y-only"
        >
          Field type: 
        </span>
        integer
      </div>
      <p>
        Total Test Results Provided by the State
      </p>
      <div
        className="detail-text detailText"
      >
        Returns 
        <code>
          null
        </code>
         if no data is available
      </div>
    </dd>
  </div>
</dl>
`;
